src/content/en/updates/2017/12/chrome-64-deprecations.md

```diff
@@ -0,0 +1,69 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: A round up of the deprecations and removals in Chrome 64 to help you plan. In this version, performance API changes, removal of support for multiple shadow roots, and removal of a webkit API.
+
+{# wf_updated_on: 2017-12-14 #}
+{# wf_published_on: 2017-12-14 #}
+{# wf_tags: deprecations,removals,chrome64 #}
+{# wf_featured_image: /web/updates/images/generic/warning.png #}
+{# wf_featured_snippet: A round up of the deprecations and removals in Chrome 64 to help you plan. In this version, performance API changes, removal of support for multiple shadow roots, and removal of a webkit API. #}
+
+# Deprecations and removals in Chrome 64 {: .page-title }
+
+{% include "web/_shared/contributors/josephmedley.html" %}
+
+In nearly every version of Chrome, we see a significant number of updates and
+improvements to the product, its performance, and also capabilities of the Web
+Platform. This article describes some of the deprecations and removals in Chrome
+64, which is in beta as of December 14. Visit the
+[deprecations page](/web/updates/tags/deprecations) for more deprecations and
+removals from this and previous versions of Chrome. This list is subject to
+change at any time.
+
+## Remove chrome.loadTimes()
+
+The `chrome.loadTimes()` method is a non-standard API that enables collecting
+loading-related metrics in real-world operations. A [Resource Timing Level
+1](https://www.w3.org/TR/resource-timing-1/) API has existed for several years
```

Comment:
Resource timing is not the only API that replaces this.
The other pieces are Navigation Timing, Paint Timing.

Reply:
Navigation timing I get and should have thought of. How is paint timing relevant?

Reply:
Paint Timing is relevant because `chrome.loadTimes()` returns a `firstPaintTime` property, and Navigation Timing is relevant because it defines values to determine all of the other properties. Resource Timing is only relevant in that it's the spec that defines [nextHopProtocol](https://www.w3.org/TR/resource-timing-2/#dom-performanceresourcetiming-nexthopprotocol) (though that value is exposed via the interface from the Navigation Timing spec).

Reply:
PTAL